### DISCUSSION TITLE
Vercel Supabase Integration with Multiple Vercel Projects

### DISCUSSION THREAD

**Discussion by robpetty** [supabase/supabase#17940] [Feature Requests]
Has anyone stood up multiple projects with the Supabase Integration on Vercel successfully?  The error:

```
Error: either NEXT_PUBLIC_SUPABASE_URL and NEXT_PUBLIC_SUPABASE_ANON_KEY env variables or supabaseUrl and supabaseKey are required!
    at (node_modules/@supabase/auth-helpers-nextjs/dist/index.js:193:10)
    at (middleware.ts:10:42)
    at (node_modules/next/dist/esm/server/web/adapter.js:163:22)
    at (node_modules/next/dist/esm/server/async-storage/request-async-storage-wrapper.js:72:23)
    at (node_modules/next/dist/esm/server/web/adapter.js:150:52)
```

The setup:

- Single account on Vercel (two apps from Github)
- Single org on Supabase (two projects in same org)
- Vercel Project A + Supabase Integration to original project in Supabase, WORKS FINE.
- Vercel Project B + Supabase Integration, FAILS.

The integration itself is setup for "All Projects".

I have not been able to find in the documentation how do you setup a second project pair using this integration. 

Locally Project B, with .env.local is working fine against Supabase Project B.

Project B was created with:
npx create-next-app@latest -e with-supabase project-b-app .

So currently rocking 13.5.4 Next.js if that matters.

**Comment by MildTomato** [comment]
@robpetty I've confirmed this is a bug. I'm moving this to issues.

**Reply by robpetty** [reply]
Appreciate it, sorry for delay in response.

**Comment by MildTomato** [comment]
Tracked bug [here](https://github.com/supabase/supabase/issues/18163) - closing discussion
